### ISSUE TITLE
 error: ‘vector’ does not name a type on Ubuntu 20.04

### ISSUE THREAD

**Issue by gijzelaerr** [lofar-astron/LOFARBeam#19]
```[ 18%] Building CXX object CMakeFiles/stationresponse.dir/AntennaField.cc.o                                                                                                                                                                                                      │g
/usr/bin/c++  -Dstationresponse_EXPORTS  -march=native -O2 -pipe -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC   -std=c++11 -o CMakeFiles/stationresponse.dir/AntennaField.cc.o -c /scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc                                     │/
In file included from /scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc:24:                                                                                                                                                                                      │b
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.h:108:13: error: ‘vector’ does not name a type                                                                                                                                                                    │u
  108 |     typedef vector<Antenna> AntennaList;                                                                                                                                                                                                                                 │i
      |             ^~~~~~                                                                                                                                                                                                                                                       │l
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.h:146:5: error: ‘AntennaList’ does not name a type; did you mean ‘AntennaField’?                                                                                                                                  │d
  146 |     AntennaList::const_iterator beginAntennae() const;                                                                                                                                                                                                                   │/
      |     ^~~~~~~~~~~                                                                                                                                                                                                                                                          │c
      |     AntennaField                                                                                                                                                                                                                                                         │a
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.h:151:5: error: ‘AntennaList’ does not name a type; did you mean ‘AntennaField’?                                                                                                                                  │s
  151 |     AntennaList::const_iterator endAntennae() const;                                                                                                                                                                                                                     │a
      |     ^~~~~~~~~~~                                                                                                                                                                                                                                                          │r
      |     AntennaField                                                                                                                                                                                                                                                         │e
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.h:247:5: error: ‘AntennaList’ does not name a type; did you mean ‘AntennaField’?                                                                                                                                  │s
  247 |     AntennaList         itsAntennae;                                                                                                                                                                                                                                     │t
      |     ^~~~~~~~~~~                                                                                                                                                                                                                                                          │/
      |     AntennaField                                                                                                                                                                                                                                                         │o
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc: In member function ‘void LOFAR::StationResponse::AntennaField::addAntenna(const LOFAR::StationResponse::AntennaField::Antenna&)’:                                                                             │b
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc:67:5: error: ‘itsAntennae’ was not declared in this scope; did you mean ‘nAntennae’?                                                                                                                           │j
   67 |     itsAntennae.push_back(antenna);                                                                                                                                                                                                                                      │-
      |     ^~~~~~~~~~~                                                                                                                                                                                                                                                          │x
      |     nAntennae                                                                                                                                                                                                                                                            │8
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc: In member function ‘size_t LOFAR::StationResponse::AntennaField::nAntennae() const’:                                                                                                                          │6
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc:72:12: error: ‘itsAntennae’ was not declared in this scope; did you mean ‘nAntennae’?                                                                                                                          │_
   72 |     return itsAntennae.size();                                                                                                                                                                                                                                           │6
      |            ^~~~~~~~~~~                                                                                                                                                                                                                                                   │4
      |            nAntennae                                                                                                                                                                                                                                                     │-
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc: In member function ‘const LOFAR::StationResponse::AntennaField::Antenna& LOFAR::StationResponse::AntennaField::antenna(size_t) const’:                                                                        │l
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc:77:12: error: ‘itsAntennae’ was not declared in this scope; did you mean ‘nAntennae’?                                                                                                                          │i
   77 |     return itsAntennae[n];                                                                                                                                                                                                                                               │n
      |            ^~~~~~~~~~~                                                                                                                                                                                                                                                   │u
      |            nAntennae                                                                                                                                                                                                                                                     │x
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc: In member function ‘LOFAR::StationResponse::AntennaField::Antenna& LOFAR::StationResponse::AntennaField::antenna(size_t)’:                                                                                    │-
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc:82:12: error: ‘itsAntennae’ was not declared in this scope; did you mean ‘nAntennae’?                                                                                                                          │g
   82 |     return itsAntennae[n];                                                                                                                                                                                                                                               │n
      |            ^~~~~~~~~~~                                                                                                                                                                                                                                                   │u
      |            nAntennae                                                                                                                                                                                                                                                     │ 
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc: At global scope:                                                                                                                                                                                              │(
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc:85:15: error: ‘AntennaList’ in ‘class LOFAR::StationResponse::AntennaField’ does not name a type                                                                                                               │m
   85 | AntennaField::AntennaList::const_iterator AntennaField::beginAntennae() const                                                                                                                                                                                            │a
      |               ^~~~~~~~~~~                                                                                                                                                                                                                                                │s
/scratch/packaging/kern/packaging/build/lofarbeam/AntennaField.cc:90:15: error: ‘AntennaList’ in ‘class LOFAR::StationResponse::AntennaField’ does not name a type                                                                                                               │t
   90 | AntennaField::AntennaList::const_iterator AntennaField::endAntennae() const                                                                                                                                                                                              │e
      |               ^~~~~~~~~~~                                                                                                                                                                                                                                                │r
make[2]: *** [CMakeFiles/stationresponse.dir/build.make:66: CMakeFiles/stationresponse.dir/AntennaField.cc.o] Error 1                                                                                                                                                            │)
make[2]: Leaving directory '/scratch/packaging/kern/packaging/build/lofarbeam/obj-x86_64-linux-gnu'                                                                                                                                                                              │ 
make[1]: *** [CMakeFiles/Makefile2:108: CMakeFiles/stationresponse.dir/all] Error 2                                                                                                                                                                                              │✗
make[1]: Leaving directory '/scratch/packaging/kern/packaging/build/lofarbeam/obj-x86_64-linux-gnu'                                                                                                                                                                              │ 
make: *** [Makefile:133: all] Error 2                                                                                                                        ```

**Comment by tammojan** [comment]
Looks like a missing `include <vector>` somewhere, I'll have a look where.

**Comment by tammojan** [comment]
Are you using the master or v4.0?

**Comment by gijzelaerr** [comment]
I only use released software :)

**Comment by tammojan** [comment]
Shall I release 4.1 with this fixed? It was fixed in d67944f
